src/lightning/pytorch/tuner/lr_finder.py

```diff
@@ -197,7 +197,9 @@ def suggestion(self, skip_begin: int = 10, skip_end: int = 1) -> Optional[float]
 
         # TODO: When computing the argmin here, and some losses are non-finite, the expected indices could be
         #   incorrectly shifted by an offset
-        min_grad = np.gradient(losses).argmin()
+        gradients = torch.diff(losses)
```

Comment:
why not [torch.gradient](https://pytorch.org/docs/stable/generated/torch.gradient.html)?

Reply:
I have updated the code, now it uses `torch.gradient`, thanks for the suggestion